docs/writing-tests/assertions-and-results.md

```diff
@@ -37,8 +37,53 @@ Inferno also implements more specific assertions to handle common cases, such as
 - Validating a FHIR Resource.
 
 Check out the [assertions API
-documentation](/inferno-core/docs/Inferno/DSL/Assertions.html) for all available
-assertions.
+documentation](/inferno-core/docs/Inferno/DSL/Assertions.html) for detailed
+information on all available assertions.
+
+### Assertion Examples
+
+```ruby
+test do
+  first_request = fhir_read(:patient, '123')
```

Comment:
> Very minor thing, but should this be named "first_response"?

Definitely not, because it's absolutely a request. The return value of those methods is the same thing you'd get from using `request` after one of those methods. If you used `response` afterwards, you'd get something else. `response` is just a Hash and it isn't very well thought out so I think we should use it as little as possible.

I don't think any of the naming options are particularly great.
- `Request` - it isn't obvious that it includes the response
- `RequestResponse` - is it the request and the response, or just the response to a request?
- `Exchange` - Is this really common nomenclature? I don't think I've ever referred to an http request/response as an exchange

Reply:
> Definitely not, because it's absolutely a request.

... given how the inferno has defined "request".  I'm not sure people picking up the DSL would have the same mental model initially.  But yeah, it is modeled as the Request class too, so this is more than just skin deep, and I don't disagree that there isn't a super-compelling alternative.  I do think Exchange would be more clear, even if its not a term that people would necessarily pull out of thin air (I'd think 'Request and Response' would be what people would think of, but thats very wordy).

In just looking through this with 'fresh eyes', the variable named seemed like a weird one, so that's what i was reacting to.  But we are consistent about it and there isn't anything substantially better.